.gitlab-ci.yml

```diff
@@ -1,12 +1,35 @@
 stages:
+  - vulnerabilities
   - test
 
-before_script:
-  - curl -sS https://getcomposer.org/installer | php
-  - php composer.phar install --no-scripts --quiet --ignore-platform-reqs
-  - cp app/config/parameters.yml.dist app/config/parameters.yml
+######################################
+# Vulnerabilities stage related jobs #
+######################################
+check php vendors:
+  stage: vulnerabilities
+  image: sumocoders/framework-php72
+  before_script:
+    - touch .env
+    - composer install --no-scripts
```

Comment:
die --quiet is niet meer nodig?

Reply:
Niet echt. `--quiet` onderdrukt output, is eerder voor in deploy, maar is hier niet relevant.